src/index.css

```diff
@@ -11,3 +11,133 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
```

Comment:
Umówmy się, że na razie taki zwykły sposób na stylowanie będzie ok. 
Aczkolwiek w praktyce nie zawsze się tak robi jak tutaj. Używamy częściej CSS modules, styled components lub JSS (dla material ui). Spróbuj kiedyś użyć np. styled components w tym projekcie. To nie będzie wymagać przepisywania wszystkich istniejących styli na to podejście.

Reply:
Ok, to na razie zostanę przy tym sposobie stylowania. W miarę postępu projektu zmienię go.